.travis.yml

```diff
@@ -21,5 +21,5 @@ script:
   - sh -x ./node_modules/patternfly-eng-release/scripts/_build.sh -o
 
 after_success:
-  - grunt build
+  - grunt build:production
```

Comment:
That would be ideal. Unfortunately, there are many changes that must occur prior to a build. Similarly, there is a bit of logic to determine what we're building; normal build, release, and alpha.